docs/planning/timeline-editing.mdx

```diff
@@ -40,40 +40,94 @@ Click on the plus button to add a new row to the timeline. A row will be added b
+- `Compact Mode`: Activities from all of the layers are packed together in a space efficient waterfall manner. This mode is useful for visualizing smaller sets of activities in a more traditional timeline view. In this mode the row will not autosize and must be manually adjusted to accommodate the current activity packing solution. This allows for a more precise adjustment of the density in the timeline.
+
+#### 2. Activity Label Visibility
+
+There are several options to control how activity labels appear in the timeline:
+
+- `On`: Always show activity labels regardless of collision.
+- `Off`: Never show activity labels.
+- `Auto`: Show activity labels when they do not collide with other labels or activities.
+
+#### 3. Directive and Span Visibility
+
+By default, both directives and spans are shown in a row. Alternatively you may choose to show only directive or only spans.
+
+#### 4. Hierarchy Root
+
+When in `Grouped` mode, the `Hierarchy` option affects the root of the type group trees. By default, `Flat` mode is selected for `Hierarchy` option. In `Flat` mode, all activities will be considered when making groups by type regardless of decomposition. This is useful when trying to understand all of the resulting activities from a simulation without needing to expand various activity decompositions. By contrast, `By Directive` mode starts this grouping at the directive level, effectively hiding any child spans until further down in the trees. This mode is useful when you are interested in examining the timeline from a decomposition perspective.
+
+#### 5. Activity Height
+
+Use the `Height` option to control the height of activities in both `Grouped` and `Compact` modes.
+
+### Understanding the Grouped Activity Tree
+
+<figure>
+  <img
+    alt="Aerie UI - Timeline Activity Tree"
+    src={timelineActivityTree}
+    style={{ maxHeight: '100%', width: '100%' }}
+  />
+  <figcaption>Figure 3: Aerie UI Timeline Activity Tree</figcaption>
+</figure>
+
+In `Grouped` mode the activity tree presented in the row header has several important indicators and actions:
+
+1. `Filter Activities by Time Window` - Automatically filter nodes in the tree when you zoom and pan around in the plan.
+2. This folder represents a group of activities whose type is `BananaNap`. Groups can contain both directives and spans.
+3. This icon represents a directive without a span.
+4. This icon represents a directive and its root span. The `parent` span has children spans which can be expanded as demonstrated here.
+5. Click on this button while hovering over a non-leaf node in the tree to select that directive/span. In this case, the `parent` row is blue since it has been selected. Note that when selecting a combined directive and span node, the span details will be preferred over the directive details. You can jump between span and directive using a button in the top right of the `Selected Activity` panel.
```

Comment:
> Note that when selecting a combined directive and span node, the span details will be preferred over the directive details. You can jump between span and directive using a button in the top right of the `Selected Activity` panel.

Nonblocking: This workflow is worth documenting somewhere related to activity directives/spans (in addition to keeping it here)

Reply:
Agreed though I don't think we actually have any doc pages that discuss viewing and editing directives/spans?

Reply:
We don't, but we should. I made a ticket for it [here](https://github.com/NASA-AMMOS/aerie-docs/issues/158), but that ticket shouldn't block these docs.

Reply:
Awesome thank you!